Makefile

```diff
@@ -4,7 +4,7 @@ GO_LDFLAGS_STATIC=-ldflags "-s -w -extldflags -static"
 default: dev
 
 dev:
-	go run ./cmd/gaia/main.go -homepath=${PWD}/tmp -dev=true -poll=true
```

Comment:
I haven't noticed that I put this in. 😊

Reply:
I guess you can leave that in. Especially when you do pipeline testing this is really helpful! 😄